proxy/hdrs/MIME.h

```diff
@@ -765,6 +768,23 @@ MIMEField::name_set(HdrHeap *heap, MIMEHdrImpl *mh, const char *name, int length
 /*-------------------------------------------------------------------------
   -------------------------------------------------------------------------*/
 
+inline bool
+MIMEField::name_validate() const
```

Comment:
It should be called name_is_valid, going with the normal naming convention of methods that return bool.